software_checklist.md

```diff
@@ -0,0 +1,141 @@
+
+### Is the code clean, generally understandable and readable and written according to good software engineering principles?
+
+- Is it modular?  Are the internal implementation details of one module hidden from other modules?
+- Commented where necessary?
+- Avoiding red flags such as very long functions, global variables, copy and pasted code, etc.?
+
+> - [x] Sufficiently addressed
+> - [ ] Some work remaining or caveats
+> - [ ] Needs to be addressed
+>
+> Code is modular, documented, low in duplication and well laid out.
+
+### Is there sufficient documentation?
+
+- Is there a readme?
+- Does the code have user documentation?
+- Does the code have developer documentation?
+- Does the code have algorithm documentation? e.g. something that describes how the model is actually simulated, or inference is performed?
+- Is all the documentation up to date?
+
+> - [ ] Sufficiently addressed
+> - [x] Some work remaining or caveats
+> - [ ] Needs to be addressed
+>
+> We have (slightly outdated) [user](README.md), [developer](https://readthedocs.org/projects/simple-network-sim/badge/?version=latest) and (slightly outdated) [algorithm](model_overview_simple_network_sim.md) documentation. A GitHub issue has been raised to address the areas of documentation that need to be updated.
+
+### Is there suitable collaboration infrastructure?
+
+- Is the code in a version-controlled repository?
+- Is there a license?
+- Is an issue tracker used?
+- Are there contribution guidelines?
+
+> - [x] Sufficiently addressed
+> - [ ] Some work remaining or caveats
+> - [ ] Needs to be addressed
+>
+> We use a [GitHub repository](https://github.com/ScottishCovidResponse/simple_network_sim), with a [BSD license](LICENSE.txt), issue tracking and [contribution guidelines](contributing.md).
+
+### Are software dependencies listed and of appropriate quality?
+
+> - [ ] Sufficiently addressed
+> - [ ] Some work remaining or caveats
+> - [x] Needs to be addressed
+>
+> Dependencies are specified in a conda [spec-file](spec-file.txt). We are investigating how best to handle `git` as a binary dependency in the conda paradigm, but this is not a major issue. We depend on the SCRC Data Pipeline and [SCRC Data Pipeline API](https://github.com/ScottishCovidResponse/data_pipeline_api) which are not yet sufficiently developed to ensure robust, reproducible operation - they are under intensive development and we expect to have this soon.
+
+### Is input and output data handled carefully?
+
+- Does the code use the data pipeline for all inputs and outputs?
+- Is the code appropriately parameterized (i.e. have hard coded parameters been removed)?
+
+> - [ ] Sufficiently addressed
+> - [ ] Some work remaining or caveats
+> - [x] Needs to be addressed
+> 
+> The SCRC Data Pipeline is not being used fully as it is under intensive development. We expect to have this soon. Code is parameterised.
```

Comment:
From the point of view of the simple network sim code this is fully addressed. Unless I'm missing something, the only issue is that we are not downloading the files from a central place yet. I would say this is at least "Soem work remaining or caveats"

Reply:
I've changed it to be more positive and reflect that the integration is in place.